### PR TITLE
[Monitor] Fixing bug #2655: the top argument of the Metrics API must be a integer instead of a double

### PR DESCRIPTION
**Pull request by gucalder**
## Description

The commit in the Swagger specs repo with the bug fix is: 34e1432d9dbe948bff63b3dda78a7f1d8bac180a

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<Description>Microsoft Azure Monitor Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
-		<Version>0.19.0-preview</Version>
+		<Version>0.19.1-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj]: Looks like a breaking change, please bump this to `0.20.0-preview`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please pull down the latest changes from upstream and run
- msbuild build.proj
- msbuild build.proj /t:Build /p:Scope=SDKs\Monitor
- generate.cmd
Please commit the artifacts generated by these steps

File: src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj (hunk 1 above)
Comment (gucalder) [reply src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj]: This is a bug fix (issue #2655 from Swagger): any call that used the top argument returned BadRequest.

**Comment by gucalder** [comment]
I followed the steps suggested above in that order, but no artifact was generated.

**Comment by dsgouda** [comment]
You might want to do a `git clean -xdf` and start over again since this will re-install the latest build tools. A txt file should be generated/modified when you go through these steps.

**Comment by gucalder** [comment]
Only a file called "_metadata/monitor_resource-manager.txt" is generated. Is that the one?

**Comment by dsgouda** [comment]
Yes, that's the one.

**Comment by gucalder** [comment]
Cool. Thanks!

**Comment by gucalder** [comment]
This is a failed test from KeyVault:

c:\workspace\NetCore-SdkCI\tools\SdkBuildTools\targets\common.targets(189,5): error MSB3073: The command "dotnet test -f netcoreapp1.1 c:\workspace\NetCore-SdkCI\src\SDKs\KeyVault\dataPlane\Microsoft.Azure.KeyVault.Tests\Microsoft.Azure.KeyVault.Tests.csproj -l trx;LogFileName=c:\workspace\NetCore-SdkCI\TestResults\netCore11\Microsoft.Azure.KeyVault.Tests.trx " exited with code 1. [c:\workspace\NetCore-SdkCI\build.proj]

**Comment by dsgouda** [comment]
@azuresdkci retest this please

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**